### PR TITLE
Feature/smittestop not active UI android

### PR DESCRIPTION
**Pull request by NC-001**
- Created new UI for "Smittestop not active" screen
- Added Danish and English texts

Screenshot_20220303-104700_Smittestop
Screenshot_20220303-104709_Smittestop

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Views/InitializerActivity.cs [hunk 1]
```diff
@@ -108,7 +108,9 @@ private void Continue()
         {
             if (PlayServicesVersionUtils.PlayServicesVersionNumberIsLargeEnough(PackageManager))
             {
-                NavigationHelper.GoToOnBoarding(this, true);
+                //NavigationHelper.GoToOnBoarding(this, true);
+                //NavigationHelper.GoToFarewellSmittestopPage(this);
+                NavigationHelper.GoToSmittestopNotActivePage(this);
```
Comment (NC-204) [inline NDB.Covid19/NDB.Covid19.Droid/Views/InitializerActivity.cs:113]: Not needed to change this, as we will change launch activity.

File: NDB.Covid19/NDB.Covid19.Droid/Resources/drawable/text_box_background.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <corners android:bottomLeftRadius="10dp" android:bottomRightRadius="10dp" android:topLeftRadius="10dp" android:topRightRadius="10dp" />
+            <stroke
```
Comment (NC-204) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/drawable/text_box_background.xml:5]: No border in figma, should it be here?
(very small thing above, figma uses border radius 8)

File: NDB.Covid19/NDB.Covid19.Droid/Views/InitializerActivity.cs (hunk 1 above)
Comment (NC-001) [reply NDB.Covid19/NDB.Covid19.Droid/Views/InitializerActivity.cs:113]: It was just for testing, forgot to remove it

**Review by NC-204 — APPROVED** [review]
💯

**Pull request merged**